packages/extras/src/lib/components/PerfMonitor/PerfMonitor.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+  import { useStage, useTask, useThrelte } from '@threlte/core'
+  import { ThreePerf } from 'three-perf'
+  import type { PerfMonitorProps } from './PerfMonitor'
+
+  type $$Props = Required<PerfMonitorProps>
+
+  export let logsPerSecond: $$Props['logsPerSecond'] = 10
+  export let showGraph: $$Props['showGraph'] = true
+  export let memory: $$Props['memory'] = true
+  export let enabled: $$Props['enabled'] = true
+  export let visible: $$Props['visible'] = true
+  export let actionToCallUI: $$Props['actionToCallUI'] = ''
+  export let guiVisible: $$Props['guiVisible'] = false
+  export let backgroundOpacity: $$Props['backgroundOpacity'] = 0.7
+  export let scale: $$Props['scale'] = 1
+  export let anchorX: $$Props['anchorX'] = 'left'
+  export let anchorY: $$Props['anchorY'] = 'top'
+
+  const { renderer, renderStage, mainStage } = useThrelte()
+
+  const perf = new ThreePerf({
+    anchorX,
+    anchorY,
+    domElement: document.body,
```

Comment:
It'd be more work but is there any merit to letting the domElement be set by the user? something like `<PerfMonitor dom={someElement} />`

All good if no